### PR TITLE
added fips specific doc for GM istio

### PR DESCRIPTION
**Pull request by christian-posta**

### CODE REVIEW

File: docs/content/setup/istio_fips.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+```shell
+/usr/local/bin/envoy  version: fa9fd362c488508a661d2ffa66e66976bb9104c3/1.15.1/Clean/RELEASE/BoringSSL-FIPS
+```
+
+To verify the control plane components, we will copy the pilot-discovery binary out of the istiod container and run [goversion](https://github.com/rsc/goversion) against it.
+
+First, copy the binary out to local disk:
+
+```shell
+kubectl cp istio-system/istiod-85ff76b4b5-47pqg:/usr/local/bin/pilot-discovery /tmp/pilot-discovery && chmod +x /tmp/pilot-discovery
```
Comment (jenshu) [inline docs/content/setup/istio_fips.md:63]:
```suggestion
kubectl cp istio-system/<istiod-pod-name>:/usr/local/bin/pilot-discovery /tmp/pilot-discovery && chmod +x /tmp/pilot-discovery
```

File: docs/content/setup/istio_fips.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+kubectl exec -it -n istio-system deploy/istio-ingressgateway -- /usr/local/bin/envoy --version
+```
+
+An appropriate response should look like this:
+
+```shell
+/usr/local/bin/envoy  version: fa9fd362c488508a661d2ffa66e66976bb9104c3/1.15.1/Clean/RELEASE/BoringSSL-FIPS
+```
+
+To verify the control plane components, we will copy the pilot-discovery binary out of the istiod container and run [goversion](https://github.com/rsc/goversion) against it.
```
Comment (jenshu) [inline docs/content/setup/istio_fips.md:58]: maybe add a line mentioning that you can install goversion via: `go get github.com/rsc/goversion`

i know i and others had trouble discovering this

File: docs/content/setup/istio_fips.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+You should see something similar to:
+
+```shell
+/tmp/pilot-discovery go1.14.12b4 (boring crypto)
+```
+
+### RSA 4096 key sizes
+
+FIPS 140-2 originally didn't allow for key sizes 4096 but that has been revised in recent addendums. With Gloo Mesh Istio's FIPS builds, you can run the control plane in strict adherence to 140-2 or in a mode that supports 4096 key sizes. For strict mode, you can append `-fipsonly` to the build tag.
```
Comment (linsun) [inline docs/content/setup/istio_fips.md:84]: would be good to call out this is also referred as relaxed FIPS?

**Review by linsun — COMMENTED** [review]
LGTM!

**Review by joekelley — APPROVED** [review]

**Pull request merged**